### ISSUE TITLE
Where did Discussions go?

### ISSUE THREAD

**Issue by Troystar23** [ShadowMario/FNF-PsychEngine#7866]
### What is your question?

Does anybody know where the discussions tab go?

**Comment by Cherif107** [comment]
shadow mario removed it because alot of kids spam discussions

**Comment by Troystar23** [comment]
that is fair I did seem to see a lot of very random and sometimes completely unnecessary questions or posts there. Does that mean in terms of asking questions we use this issues section?

**Comment by ActualeAnEggDog** [comment]
it can be on 1th april - day of jokes

**Comment by Troystar23** [comment]
that's true, it could be. I just saw another question talking about that, I completely forgot today was the first.

**Comment by Razthefunni** [comment]
its prob an april fools joke or he was actually serious because brainless kids wouldn't stop spamming and was just only asking for lua shit

**Comment by Troystar23** [comment]
yeah imagine asking about lua stuff haha..... :(

**Comment by Razthefunni** [comment]
don't get me wrong you can ask for lua for something but there was just wayyyy to many people asking for it for like the custom menu and other stuff thats impossible for lua and it was kinda getting annoying with the amount of people  i was mainly asking for source AND lua stuff i do both

**Comment by Troystar23** [comment]
oh I agree most stuff should be down source code I didnt think lua was meant to be used for everything I dont know why people would assume that. I just like using it for small functions like events and crap, its quick and easy and I can do it live.

**Comment by Razthefunni** [comment]
same if im too lazy to do it on source ill just do it in lua

**Comment by Troystar23** [comment]
exactly lua is just fun like that

**Comment by Razthefunni** [comment]
![image](https://user-images.githubusercontent.com/98192860/161272439-1d1a7520-fe3e-410a-bd53-254822e8a4a6.png)
honsetly now i don't think its an april fools

**Comment by Troystar23** [comment]
oh lmao well i guess that answers it

**Comment by JacyCoboBoy** [comment]
What can we do now? search up YouTube videos of psych engine tutorials???

**Comment by RealReal334** [comment]
Bruh it was really good place to seek help about lua, i admitted that annoying people spam it 
but if it's gone Where would those Precious Ideas go????

**Comment by im557** [comment]
> Bruh it was really good place to seek help about lua, i admitted that annoying people spam it but if it's gone Where would those Precious Ideas go????

to the grave :)

**Comment by megasssdev** [comment]
> Bruh it was really good place to seek help about lua, i admitted that annoying people spam it but if it's gone Where would those Precious Ideas go????

on the discord.

**Comment by Razthefunni** [comment]
> What can we do now? search up YouTube videos of psych engine tutorials???

the only psych engine tutorials out there are just the average how to add a custom character stage music dialogue etc
